src/main/java/org/folio/rmapi/model/CustomerResources.java

```diff
@@ -0,0 +1,160 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CustomerResources {
+
+  @JsonProperty("titleId")
+  private String titleId;
+
+  @JsonProperty("packageId")
+  private Integer packageId;
+
+  @JsonProperty("packageName")
+  private String packageName;
+
+  @JsonProperty("isPackageCustom")
+  private Boolean isPackageCustom;
+
+  @JsonProperty("vendorId")
+  private Integer vendorId;
+
+  @JsonProperty("vendorName")
```

Comment:
Both these "vendorId" and "vendorName" can be removed. These will always be redundant with "providerId" and "providerName" and UI will only recognize the latter. Ruby code should be cleaned up to not give "vendor*" in its responses but we will deal with that cleaning part here.

Reply:
Believe vendorId and vendorName are ok in this context since this as how the RM API response refers to these fields which in turn get mapped to provider

Reply:
You are right Carole, I just realized that this class is mapping RM API json props.